### ISSUE TITLE
Support SyncBatchNorm in Quantization aware training

### ISSUE THREAD

**Issue by raghuramank100** [pytorch/pytorch#33646]
Sync Batch Norm is used for training CV models for detection/segmentation use cases in a distributed setting. We will need to create a qat.ConvSyncBn2d module to fold this module for quantization aware training and ensure that the properties of sync bn are preserved.

cc @jerryzh168 @jianyuh @dzhulgakov @raghuramank100 @jamesr66a

**Comment by supriyar** [comment]
@vkuzo I believe this is now supported? Can we close the issue?

**Comment by vkuzo** [comment]
Yep, https://github.com/pytorch/pytorch/pull/38478 fixes this, will be out with v1.6.
